### ISSUE TITLE
Broken webfonts: ‘url([object Module])’ in @font-face

### ISSUE THREAD

**Issue by Clockmaker** [apache/incubator-superset#10786]
Hi, 
I was updating superset to 0.37 and I noticed that all the webfonts are broken

On firefox 78 there is also this warning:
`Unknown descriptor ‘url([object Module])’ in @font-face rule.  Skipped to next declaration.`

I gave a look at the css generated by less-loader and indeed all the font-face are broken starting from 0.37 onwards.

![2020-09-03-210245_3840x1080_scrot](https://user-images.githubusercontent.com/117092/92157303-8b389600-ee2a-11ea-9b30-5efebc717857.png)
#### How to reproduce the bug

````bash
docker run -it preset/superset:0.37 /bin/bash
````
inside the container:
````bash
grep -r "object Module" /app/superset/static/assets/ --color="auto"
````
I also pulled the `38da552a57b9be999fb36b2cf7ae1c9231a010eb` (14 days old) with the same result.

### Checklist

- [ ] ~~I have checked the superset logs for python stacktraces and included it here as text if there are any.~~
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

I compared `webpack.config`, `package.json` and `Dockerfile` with the 0.36 but I fail to see any possible breaking change.

#### Possible solution
I tried to add `esModule: false` to [less-loader](https://github.com/apache/incubator-superset/blob/master/superset-frontend/webpack.config.js#L295) and it seems to solve the issue (partially, because I still have some warnings).

Thanks.

**Comment by JunlinC** [comment]
@rusackas

**Comment by rusackas** [comment]
This was resolved with #11718 and #11724
